### DISCUSSION TITLE
Incorrect redirects with multiple language sites

### DISCUSSION THREAD

**Discussion by mihob** [craftcms/commerce#3312] [Q&A]
I have installed the demo template and two pages for different languages. The base urls are https://www.example.com/de and https://www.example.com/en respectively.

Now, however, the redirects are generated incorrectly. For example, after saving an address in the checkout process, I am redirected to the page https://www.example.com/de/de/shop/customer/addresses, which does not exist. Have I configured something incorrectly? Or is this a bug?

This happens with the current version of Craft CMS and Craft commerce.

**Comment by lukeholder** [comment]
Can you share your redirect input html/twig? Thanks

**Reply by mihob** [reply]
Sure, this was generated by the twig helper `{{ redirectInput(redirect ?: 'shop/customer/addresses') }}`

`<input type="hidden" name="redirect" value="cd7d4dc45be276c9b039d1ee0d06527d8f0bd1e0ea44b21aa69eaed862dc6621de/shop/checkout/addresses">`
